guides/configure-oE-devEnviron-ubuntu-vm/How-to-install-openEQUELLA-Development-Environment-in-Virtual-Box-Ubuntu-OS.md

```diff
@@ -0,0 +1,626 @@
+
+`$ sudo apt update`</br>
+`$ sudo apt -y install vim bash-completion wget`</br>
+`$ sudo apt -y upgrade`</br>
+
+3. You must reboot whenever you upgrade.
+
+`$ sudo reboot`
+
+4. Now add the repository.
+
+`$ wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -`
+
+5. Install postgresql
+
+`$ sudo apt update`</br>
+`$ sudo apt -y install postgresql-12 postgresql-client-12 postgresql-contrib-12`
+
+6. Check to see that you can connect to postgresql using psql.
+
+`$ sudo -u postgres psql -c "SELECT version();"`
+
+![check version](images/pg%2001%20check%20version.png)
+
+7. Set the password for the use postgres. To do this follow the commands below.
+
+`$ sudo -u postgres psql postgres`</br>
+`# \password postgres`
+
+The output will look like this.
+
+![change postgres password](images/postgresql/01%20change%20postgres%20password.png)
+
+8. Type “exit” to escape root.
+
+### <a id="user-content-install-pgadmin" class="anchor" aria-hidden="true" href="#install-pgadmin"></a>Install Pgadmin4
+
+1. Now install the PostgreSQL admin tool. All of the command will be provided below. The website from where these commands were taken is found here:  [Click here to view Pgadmin Install for Ubuntu 20.40.4](https://yallalabs.com/linux/how-to-install-pgadmin4-ubuntu-20-04/)
+
+2. Run the command below to install the pgAdmin4 key.
+
+`$ curl https://www.pgadmin.org/static/packages_pgadmin_org.pub | sudo apt-key add`
+
+3. pgAdmin4 requires you to add an external repository.
+
+`$ sudo sh -c 'echo "deb https://ftp.postgresql.org/pub/pgadmin/pgadmin4/apt/$(lsb_release -cs) pgadmin4 main" > /etc/apt/sources.list.d/pgadmin4.list && apt update'`
+
+4. Now install pgAdmin as server mode.
+
+`$ sudo apt install pgadmin4-web`
+
+5. Now configure pgAdmin by running the command below.
+
+`$ sudo /usr/pgadmin4/bin/setup-web.sh`
+
+`The output should look like what you see below. You will be prompted for an email address and password. You will also be prompted to restart the apache server. `
+
+`Setting up pgAdmin 4 in web mode on a Debian platform...`
+`Creating configuration database...`
+`NOTE: Configuring authentication for SERVER mode.`
+
+`Enter the email address and password to use for the initial pgAdmin user account:`
+
+`Email address: salmon@salmon.com`
+`Password:`
+`Retype password:`
+`pgAdmin 4 - Application Initialisation`
+`======================================`
+
+`Creating storage and log directories...`
+`We can now configure the Apache Web server for you. This involves enabling the wsgi module and configuring the pgAdmin 4 application to mount at /pgadmin4. Do you wish to continue (y/n)? y`
+`The Apache web server is running. A restart is required for the pgAdmin 4 installation to complete. Would you like to continue (y/n)? y`
+`Apache successfully restarted. You can now start using pgAdmin 4 in web mode`
```

Comment:
Could this be made a single, multi-line code block?
https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks